orttraining/orttraining/training_api/module.cc

```diff
@@ -168,6 +177,14 @@ Module::Module(const std::string& train_model_path_or_bytes,
   }
 }
 
+size_t Module::GetTrainModeOutputCount() const noexcept {
```

Comment:
so the only use of this is to set the size of output vector, since its mutable we can do without this function right? Why increase steps for user?